src/Symfony/Component/Messenger/Worker.php

```diff
@@ -278,6 +291,20 @@ public function stop(): void
         $this->shouldStop = true;
     }
 
+    public function keepalive(): void
+    {
+        foreach ($this->keepalives as $message) {
+            [$transportName, $envelope] = $this->keepalives[$message];
+
+            if (!$this->receivers[$transportName] instanceof KeepaliveReceiverInterface) {
+                throw new RuntimeException(sprintf('Receiver for "%s" does not implement "%s".', $transportName, KeepaliveReceiverInterface::class));
+            }
+
+            $this->logger?->info('Sending keepalive request.', ['transport' => $transportName]);
```

Comment:
please include the message ID :') ref #53808

Reply:
Good idea, done

Reply:
it's not an idea